### PR TITLE
Round Length Timer

### PR DESCRIPTION
**Pull request by 0marA**
The host can set the round length from the create page and the players will see the countdown until the end of the round. There is also a submission indicator for the judge to see who has and hasn't submitted. There are also minor styling and navigation bug fixes.

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Home = () => {
       </h1>
 
       <h4 className="home-sub-heading">
-        A party game <br /> for horrible machines.
+        A party game <br /> for horrible robots.
```
Comment (Juicestus) [inline client/src/pages/Home.js:11]: Nice 😎

File: client/src/index.js [hunk 2]
```diff
@@ -10,9 +10,11 @@ import Lobby from "./pages/Lobby";
 import Prompt from "./pages/Prompt";
 import Vote from "./pages/Vote";
 import Results from "./pages/Results";
-import 'bootstrap/dist/css/bootstrap.css';
+import "bootstrap/dist/css/bootstrap.css";
 
-ReactDOM.render(
+const root = ReactDOM.createRoot(document.getElementById("root"));
+
+root.render(
```
Comment (Juicestus) [inline client/src/index.js:17]: Thank god for fixing this, I am way too lazy.

File: client/src/pages/Prompt.js [hunk 3]
```diff
@@ -6,51 +6,120 @@ import {
...
+          </h1>
+
+          <div className="prompt-timer">
+            <Timer roundLength={roundLength} startTime={startTime} />
+          </div>
+
+          {Object.values(players)
+            .sort()
+            .map((player, index) => {
+              return player.name === host ? (
```
Comment (Juicestus) [inline client/src/pages/Prompt.js:76]: Why are you ignoring the host here?

File: client/src/pages/Create.js [hunk 4]
```diff
@@ -46,8 +48,39 @@ const Create = () => {
           className="create-join-big-input-box"
           onChange={bindInput(setName)}
         />
-        <br></br>
-        <br></br>
+        <div className="create-set-round-length">
```
Comment (Juicestus) [inline client/src/pages/Create.js:51]: Is this ugly? Knowing you it probably is 💀 .

**Review by Juicestus — COMMENTED** [review]
Just respond to the comment(s).

File: client/src/pages/Prompt.js (hunk 3 above)
Comment (0marA) [reply client/src/pages/Prompt.js:76]: This indicates who has and hasn't submitted yet. The host can't submit so they're excluded from the list

**Review by Juicestus — APPROVED** [review]
LGTM👍

**Pull request merged**